docs/java_cli_for_managedgateway.rst

```diff
@@ -145,8 +145,8 @@ The gateway can invoke sendDeviceManageRequest() method to make the attached dev
 
 .. code:: java
 
-	managedGateway.sendGatewayManageRequest(typeId, deviceId, lifetime, true, true);
-	
+	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);
```

Comment:
it should be phrased like one need to specify lifetime, device/firmware support parameters along with the details of the attached device.